install/resources/upgrade35.php

```diff
@@ -164,12 +164,12 @@ function upgrade35_dbchanges4()
 	echo "<p>Adding index files to attachment directories...</p>";
 	flush();
 
-	$dir = @opendir('../'.$mybb->settings['uploadspath']);
+	$dir = @opendir($mybb->settings['uploadspath']);
```

Comment:
If the user is using a relative path what will happen here as well? Since the upgrade is ran from a sub folder.

Reply:
Similarly as above, any relative path is converted to an absolute path just after settings are loaded, this time in `install/upgrade.php`:

https://github.com/mybb/mybb/pull/4342/files#diff-43f8477f93fbadcef16f90e9fbffd75dd83dc143d06fe9c7304ffb35e8c03794R121-R125

So, in all cases, we can rely on the path being absolute (or, at least, that's the intent - the changes haven't been carefully tested).